stablehlo/reference/Ops.cpp

```diff
@@ -1105,10 +1105,10 @@ Tensor evalCollectiveBroadcastOp(
   if (channelId <= 0) processGroups = process->crossReplica(replicaGroups);
   if (channelId > 0) processGroups = process->crossPartition(replicaGroups);
 
-  for (auto processGroup : processGroups)
-    if (llvm::find(processGroup, process->getId()) != processGroup.end())
-      return process->rendezvous(processGroup, channelId, operand)
-          .lookup(processGroup[0]);
+  auto processGroup = processGroups.findGroup(process->getId());
+  if (processGroup)
```

Comment:
Nit: `if (auto processGroup = ...) ...` would be slightly cleaner, since `processGroup` only needs to be in scope inside the `if`.

Reply:
Ack. This pattern exists throughout the file, and I plan to create a PR to clean them up later.